From: Manoj Srivastava <srivasta@debian.org>
Date: Thu, 3 Sep 2009 23:34:34 -0500
Subject: [PATCH] [topic--debian]: Fix Wrong path for interpreter
Bug-Debian: https://bugs.debian.org/544536

Mostly tests and examples.

Signed-off-by: Manoj Srivastava <srivasta@debian.org>

--- a/eg/benchmarks/simple
+++ b/eg/benchmarks/simple
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 ###########################################
 # Log4perl Benchmarks
 # Mike Schilli, 2008 (m@perlmeister.com)
--- a/eg/newsyslog-test
+++ b/eg/newsyslog-test
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 ###########################################
 # newsyslog-test
 # Mike Schilli, 200t (m@perlmeister.com)
--- a/t/027Watch4.t
+++ b/t/027Watch4.t
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 BEGIN { 
     if($ENV{INTERNAL_DEBUG}) {
