From: Manoj Srivastava <srivasta@debian.org>
Date: Sat, 31 Oct 2009 01:31:53 -0500
Subject: [PATCH] [topic--man-fix]: Fixed whatis info for man page

Signed-off-by: Manoj Srivastava <srivasta@acm.org>

--- a/lib/Log/Log4perl/Layout/PatternLayout/Multiline.pm
+++ b/lib/Log/Log4perl/Layout/PatternLayout/Multiline.pm
@@ -30,7 +30,7 @@ __END__
 
 =head1 NAME
 
-    Log::Log4perl::Layout::PatternLayout::Multiline
+Log::Log4perl::Layout::PatternLayout::Multiline - Multiline Pattern Layout
 
 =head1 SYNOPSIS
 
